app/src/main/java/org/akvo/flow/util/logging/RavenEventBuilderHelper.java

```diff
@@ -37,6 +41,10 @@ public RavenEventBuilderHelper(Map<String, String> tags) {
     @Override
     public void helpBuildingEvent(EventBuilder eventBuilder) {
         eventBuilder.withSdkName(RavenEnvironment.SDK_NAME + ":android");
+        String release = tags.get(VERSION_NAME_TAG_KEY);
```

Comment:
I was just wondering if perhaps it makes sense to include this in the `for` loop below instead
i.e. `if (VERSION_NAME_TAG_KEY.equals()) {...}`

One other issue is whether you want the version name to be included both in the release property and as a tag.